daml-assistant/integration-tests/src/DA/Daml/Assistant/IntegrationTests.hs

```diff
@@ -157,8 +157,8 @@ packagingTests = testGroup "packaging"
           ]
         writeFileUTF8 (tmpDir </> "proj" </> "A.daml") $ unlines
           [ "module A where"
-          , "import Main (setup)"
-          , "setup' = setup"
+          , "import Main (Asset)"
```

Comment:
The `Script` type is apparently too much for `data-dependencies` to handle. This isn’t actually what we want to test here so using a type synynym as a replacement is good enough.